tests/python/gaia-ui-tests/gaiatest/apps/system/app.py

```diff
@@ -101,6 +101,16 @@ def wait_for_app_update_to_clear(self):
         Wait(self.marionette).until(lambda m: element.location['y'] == (0 - element.size['height']))
 
     @property
+    def screen_width(self):
+        self.marionette.switch_to_frame()
+        return self.marionette.find_element(*self._screen_locator).rect['width']
+
```

Comment:
Well, nearly every method in this class calls `switch_to_frame()`, so I'd prefer to keep the consistency. What do you think @mwargers ?